docs/docs.md

```diff
@@ -0,0 +1,585 @@
+# `/event/{eventKey}/info`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  webcasts: {
+    type: "twitch" | "youtube"
+    url: string
+  }[]
+  location: {
+    name: string
+    lat: number
+    lon: number
+  }
+  name: string
+  district?: string
+  week?: number
+  // UTC date
+  startDate: string
+  // UTC date
+  endDate: string
+}
+```
+
+# `/event/{eventKey}/match/{matchKey}/info`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
```

Comment:
There is no way to get score from a match here, is that intentional?

Reply:
At this point the frontend UI doesn't display match scores, but that will probably be added at some point so I added it to the API